modules/migration-editing-pvs-in-migplan.adoc

```diff
@@ -6,15 +6,30 @@
 [id="migration-editing-pvs-in-migplan_{context}"]
 = Editing persistent volume attributes
 
-You can edit the storage class and the access mode of persistent volumes (PVs) in the `MigPlan` custom resource (CR) after the PVs have been discovered by the `MigrationController` CR.
+After you create a `MigPlan` custom resource (CR), the `MigrationController` CR discovers the persistent volumes (PVs). The `spec.persistentVolumes` block and the `status.destStorageClasses` block are added to the `MigPlan` CR.
+
+You can edit the values in the `spec.persistentVolumes.selection` block. If you change values outside this block, they are overwritten when the `MigPlan` CR is reconciled by the `MigrationController` CR.
+
+[NOTE]
+====
+The default value for the `spec.persistentVolumes.selection.storageClass` parameter is determined by the following logic:
+
+. If the source cluster PV is Gluster or NFS, the default is either `cephfs`, for `accessMode: ReadWriteMany`, or `cephrbd`, for `accessMode: ReadWriteOnce`.
```

Comment:
@apinnick I don't understand it 
> If **the source cluster** PV is Gluster or NFS, the **default** is either `cephfs ..

Reply:
It is in Scott's PR: https://github.com/konveyor/mig-operator/pull/767/files

docs/usage/MigPlanPVSelection.md, line 44

Reply:
@sseago Can you confirm this? Will also ping you in Slack.

Reply:
@XinRedhat The purpose behind this was to facilitate OCS users. For 3 clusters (either with OCS, using gluster, or without,  using NFS), the intent is to default to OCS4 ceph storage classes *if* they're available. If they're not available, we can't default to them, so we fall back to the other defaults (same provisioner if available, otherwise the default storageclass for the dest/target cluster).